pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -724,6 +753,37 @@ def make_parser(parser: argparse.ArgumentParser):
             "needed if you want to build other packages that depend on this one."
         ),
     )
+    parser.add_argument(
+        "--force",
+        action="store_true",
+        help=(
+            "Force rebuild of package regardless of whether it appears to have been updated"
+        ),
+    )
+    parser.add_argument(
+        "--continue",
+        type=int,
+        nargs="?",
+        dest="continue_from",
+        const="capture",
+        help=(
+            dedent(
+                """
+                Continue a build from the middle. For debugging. Implies "--force".
+                Possible arguments:
+
+                    'capture' : redo capture step and replay step (but don't prepare
+                                sources again, use existing source directory). Good for debug
+                                builds with hand-modified sources. `--continue` with no argument
+                                has the same effect.
+
+                    'replay' : Don't redo the capture step but redo the replay step.
+
+                    'replay:15' : replay the capture step starting with the 15th compile command (any integer works)
```

Comment:
Can you mention this in the docs https://pyodide.org/en/stable/development/new-packages.html ? Maybe under some separate section on debugging build failures..